src/generation.rs

```diff
@@ -260,6 +260,10 @@ fn find_bounding_box(input: &InputFile) -> Aabb3<f32> {
     }
 
     stream.for_each(|p: &Point| {
+        if num_points == 0 {
+            let p3 = Point3::from_vec(p.position);
+            bounding_box = Aabb3::new(p3, p3);
```

Comment:
It works, but how about initializing the bounding box with the first two points and then entering the loop? so we can spare the integer comparison during the loop and remove the erroneous initialization in line 256.

Reply:
i noticed you mentioned the same thing on the description!

Reply:
This seems fine to me. Bounding box calculation is not the time critical piece, and the branch will run very hot in branch predcition quickly, so performance will not suffer either. Let's have it that way.